### PR TITLE
(Docs): Added Call-for-ideas Blog post

### PR DESCRIPTION
**Pull request by Abhijay007**
# Summary of Changes

Added an announcement post for call mentors/contributors to add ideas in the project idealist for GSoC'2024

## Checklist

- [x] I have read and followed the [project's contribution guidelines](https://github.com/secure-systems-lab/lab-guidelines/blob/master/dev-workflow.md), including code style and commit message conventions.
- [x] My code is well-documented, and I've updated relevant documentation.
- [x] I have tested these changes on my local environment.
- [x] All tests pass, and there are no new linting errors.
- [x] I have reviewed and proofread my code and the changes.
- [x] The branch is up-to-date with the base branch.

### CODE REVIEW

**Comment by Abhijay007** [comment]
As discussed in today's website call 

cc @jhdalek55, @hexsecs, @iramcdonald

File: blog/2024-01-10-GSoC-blog/index.md [hunk 1]
```diff
@@ -47,13 +47,13 @@ The objective of this project is to deconstruct the current website layout and d
 
 During our first meeting, we needed to make an important decision: whether to stick with the existing technology, which is Jekyll, or migrate the site to a new framework, Docusaurus. After a discussion, we decided to go with Docusaurus as it is quite easy to maintain and comparatively better than Jekyll. The mentors assigned me the task of testing the existing content of the site with Docusaurus so that we can determine if using this program would be feasible. So, I created a demo version of the site and shared it with the mentors. After review, both the mentors and other community members reviewed our decision to go with Docusaurus as our main framework to build the site. The mentors then suggested that I move forward and prepare a new design layout for the site.
 
-### Site New Layout 📐
+### New Site Layou 📐
```
Comment (jhdalek55) [inline blog/2024-01-10-GSoC-blog/index.md:50]:
```suggestion
### New Site Layou 📐
```
```suggestion
### New Site Layout📐
```

**Review by jhdalek55 — CHANGES_REQUESTED** [review]
Just fix new typo

**Comment by Abhijay007** [comment]
> Just fix new typo

@jhdalek55 have you looked into the call-for-ideas blog as well?

**Review by jhdalek55 — APPROVED** [review]

**Comment by Abhijay007** [comment]
@jhdalek55 have you looked into the call-for-ideas blog as well? here it is : https://github.com/uptane/uptane.github.io/pull/172/files#diff-c262a09f4771d7f99ba7d2df352d2edeeb52e3b5eec04fae735f7ca5294bfa9c

File: blog/2024-01-12-callForIdeas/index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  - Abhij
+---
+
+![](callForIdeasBanner.png)
+
+# Title: Exciting Opportunities Await: Uptane Calls for GSoC 2024 Idea List Submissions!
+
+Greetings, Uptane community 👋!
+
+As the dawn of Google Summer of Code (GSoC) 2024 ☀️ approaches, we're thrilled to announce a call for idea list submissions. Jon Oster has already put forth two compelling proposals, and we're eager to expand this list with your innovative ideas. Before diving into the details of Jon's proposals, let's encourage everyone to participate in the discussion and contribute fresh perspectives on our [GitHub discussion board](https://github.com/uptane/uptane.github.io/discussions/166).
```
Comment (jhdalek55) [inline blog/2024-01-12-callForIdeas/index.md:13]:
```suggestion
As Google Summer of Code (GSoC) 2024 ☀️ approaches, we're thrilled to announce an open call for possible Uptane projects. One of our community members, Jon Oster, has already put forth two compelling proposals, and we're eager to expand this list with your innovative ideas. Before diving into the details of Jon's proposals, let's encourage everyone to participate in the discussion and contribute fresh perspectives on our [GitHub discussion board](https://github.com/uptane/uptane.github.io/discussions/166).
```

File: blog/2024-01-12-callForIdeas/index.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+Now, let's explore the exciting projects proposed by Jon Oster:
```
Comment (jhdalek55) [inline blog/2024-01-12-callForIdeas/index.md:15]:
```suggestion
Here are the current projects proposed by Jon Oster:
```

File: blog/2024-01-12-callForIdeas/index.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+## Proposal 1: Integrating aktualizr with Popular Embedded Software Updaters 🔄
+
+Aktualizr, a Uptane client written in C++, focuses on embedded Linux systems. Uptane's core functionality is vital for securing and validating software updates in highly security-sensitive and safety-critical systems. However, aktualizr currently supports only OSTree as a method for installing updates.
+
+This GSoC project aims to enhance aktualizr by implementing support for an A/B partition-based update method. Swupdate or RAUC, two popular open-source software updaters, are the targeted integration points. While libaktualizr already supports additional package managers, collaboration with the maintainers of Swupdate or RAUC would be ideal.
```
Comment (jhdalek55) [inline blog/2024-01-12-callForIdeas/index.md:21]:
```suggestion
This GSoC project aims to enhance aktualizr by implementing support for an A/B partition-based update method. Swupdate or RAUC, two popular open-source software updaters, are the targeted integration points. While libaktualizr already supports additional package managers, our ideal solution would be a collaboration with the maintainers of Swupdate or RAUC.
```

File: blog/2024-01-12-callForIdeas/index.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+**Key skills required:** Systems-level programming in C++
+
+**Potential mentors:** Jon Oster, Patti Vacek, Phil Wise, Rogerio Borin, Phil Lapczynski, Person-to-be-named-later from SWUpdate or RAUC project.
+
+## Proposal 2: Creating an Open-Source Web UI for OTA Community Edition  🌐
+
+Uptane's project already boasts open-source implementations of core back-end services. However, setting up and using these services can be challenging. OTA Community Edition provides a simplified, docker-based deployment method, but lacks a crucial component - a user-friendly UI.
```
Comment (jhdalek55) [inline blog/2024-01-12-callForIdeas/index.md:29]: If we are talking about the project Docker, it should be capitalized.

**Review by jhdalek55 — CHANGES_REQUESTED** [review]
Note corrections

**Comment by jhdalek55** [comment]
Done

**Review by jhdalek55 — APPROVED** [review]
Done.

**Pull request merged**